### DISCUSSION TITLE
Mandatory IP or domain for CDN settings آیپی یا دامنه اجباری برای تنظیمات سی دی ان

### DISCUSSION THREAD

**Discussion by R30na** [hiddify/Hiddify-Manager#127] [Q&A سوالات رایج] [answered]
اخیرا قسمت آیپی یا دامنه اجباری در تنظیمات دامنه ها اضافه شده 

![Screenshot 2023-02-12 at 21 38 20](https://user-images.githubusercontent.com/13730082/218335864-6ebdeaac-e838-432e-aa25-0787e2eb9d39.png)

من دامین رو روی CDN کلودفلر ثبت کردم و از دامین [tgju.org](https://tgju.org/)  که میدونم از کلودفلر استفاده میکنه و توی ایران فیلتر نیست برای اتصال استفاده میکنم. به شکل تنظیمات زیر از host  و sni برای معرفی دامنه خودم استفاده میکنم: 

vless://[id]@tgju.org:443?security=tls&sni=[domain]&type=ws&path=[path]&host=[domain]&encryption=none&fingerprint=chrome&headerType=None#tls_WS_CDN_vless

حتی برای اتصال میتونم از دامین های دیگه ای که از سی دی ان کلودفلر استفاده میکنن هم استفاده کنم. 

حالا این تنظیمات برای خارج از ایران به درستی و سرعت بالا کار میکنه و برای کسایی که تو ایران هستن برای بعضی ها خوب کار میکنه ولی برای یه عده اصلا کار نمیکنه. حالا سوال من اینه چطور ممکنه برای عده ای کار نکنه در حالی که دسترسی به دامین  tgju.org  برای ایران برقرار هست؟

**Comment by hiddify-com** [comment, marked as answer]
علت اینه که فیلترچی یه سری دامنه وایت لیست داره
یعنی اون سایت ها اصن موارد دیگه مثل ای پی براشون بررسی نمیشه

دامین فرانتینگ به این هدف هست که کامل اون سایت وایت لیست را شبیه سازی کنه

**Reply by akv79** [reply]
هیچکدوم از آیتم های این وایت لیست رو میتونید معرفی کنید که استفاده کنیم

**Reply by R30na** [reply]
خب منم از دامنه [tgju.org](https://tgju.org/) استفاده کردم دیگه. مگه این دامنه تو وایت لیست نیست؟

**Reply by hiddify-com** [reply]
چیزی که فیلترچی چک میکنه sni هست
